### PR TITLE
feat(clevercloud): add clevercloud secrets

### PR DESCRIPTION
**Pull request by ykatchou**
I did add the clevercloud_sso_salt secrets across the deployment chain.
Related to https://github.com/octopize/avatar/pull/1368

### CODE REVIEW

File: helm-chart/templates/api-secrets.yaml [hunk 1]
```diff
@@ -18,4 +18,5 @@ data:
   db_user: {{ .Values.dbUsername | b64enc | quote }}
   db_password: {{ .Values.dbPassword | b64enc | quote }}
   aws_mail_account_access_key_id: {{ .Values.api.awsMailAccountAccessKeyId | b64enc | quote }}
-  aws_mail_account_secret_access_key: {{ .Values.api.awsMailAccountSecretAccessKey | b64enc | quote }}
\ No newline at end of file
+  aws_mail_account_secret_access_key: {{ .Values.api.awsMailAccountSecretAccessKey | b64enc | quote }}
+  clevercloud_sso_sal: {{ .Values.api.clevercloudSsoSalt | b64enc | quote }}
```
Comment (tcrasset) [inline helm-chart/templates/api-secrets.yaml]:
```suggestion
  clevercloud_sso_salt: {{ .Values.api.clevercloudSsoSalt | b64enc | quote }}
```

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -95,6 +97,7 @@ services:
       - file_encryption_key
       - db_name
       - db_user
+      - clevercloud_sso_salt
```
Comment (tcrasset) [inline docker/docker-compose.yml]: I don't think it's necessary to have it in the worker

File: docker/Makefile [hunk 3]
```diff
@@ -67,6 +67,9 @@ $(SECRETS_DIR)/aws_mail_account_secret_access_key:
 	mkdir -p $(SECRETS_DIR)
 	echo "" > $@
 
+$(SECRETS_DIR)/clevercloud_sso_salt:
+	mkdir -p $(SECRETS_DIR)
+	python3 -c "import secrets; print(secrets.token_hex())" > $@
```
Comment (tcrasset) [inline docker/Makefile:73]: You have to add it in `secrets-with-username-auth` and `secrets-with-email-auth` too.

**Review by tcrasset — CHANGES_REQUESTED** [review]
Very nice that you've thought about adding it in the helm template + docker-compose!

Small changes needed though!

**Comment by ykatchou** [comment]
Hello, i did take all your comments into account. I did also rebase.

File: helm-chart/templates/_helpers.tpl [hunk 4]
```diff
@@ -122,11 +122,16 @@ Define the default app env variables
                 configMapKeyRef:
                   name: avatar-config
                   key: MAX_ALLOWED_LINES_PER_DATASET
-            - name: LOG_LEVEL
+			- name: LOG_LEVEL
```
Comment (tcrasset) [inline helm-chart/templates/_helpers.tpl]: can you run  `make render-helm` (or `make lint`) to make sure the helm template works as you switched to tabs here? Makes me think that we should have a `make lint-fix`

**Review by tcrasset — APPROVED** [review]

**Pull request merged**